GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1502,12 +1501,14 @@ void DoCommit()
 
                     if (AppSettings.CloseCommitDialogAfterCommit)
                     {
+                        Message.Text = string.Empty; // Message.Text has been used and stored
```

Comment:
Why not move the Commit Message handling to before this `if` (instead of repeating the code in all branches)?

Reply:
You are right, thanks!